CONTRIBUTING.md

```diff
@@ -95,3 +95,13 @@ Entrypoint of CICD is the file .gitlab-ci.yml.
 
 There you can find the full definition of the CI/CD pipeline.
 For more details see the [CICD wiki in our Dev Guide](https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD)
+
+# How to make a new VDK release
+
+To make a new stable release, follow these steps:
```

Comment:
let's call it public release (minor or major version release). All releases should be stable as long as all tests have passed.

Reply:
noted